base/markdown/render/html.jl

```diff
@@ -8,6 +8,20 @@ function withtag(f, io, tag)
     print(io, "</$tag>")
 end
 
+const _htmlescape_chars = Dict('<'=>"&lt;", '>'=>"&gt;",
+                               '"'=>"&quot;", '\''=>"&#39",
```

Comment:
That would avoid errors missing semicolon errors (the bane of a programmers life, of course).